org.alloytools.alloy.dash/.classpath

```diff
@@ -6,7 +6,6 @@
 			<attribute name="test" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry combineaccessrules="false" kind="src" path="/org.apache.velocity"/>
```

Comment:
What was the intention of this line?

Reply:
This fixes a build error I was encountering